src/component/Tooltip.tsx

```diff
@@ -33,30 +37,25 @@ function renderContent<TValue extends ValueType, TName extends NameType>(
   return <DefaultTooltipContent {...props} />;
 }
 
-export type TooltipProps<TValue extends ValueType, TName extends NameType> = DefaultProps<TValue, TName> & {
+export type TooltipProps<TValue extends ValueType, TName extends NameType> = ToltipContentProps<TValue, TName> & {
```

Comment:
Changes to props:

1. "Sort lines alphabetically"
2. `viewBox` uses (identical) imported type instead of inline.